ChattoAdditions/Source/Chat Items/BaseMessage/BaseMessagePresenter.swift

```diff
@@ -101,6 +103,8 @@ open class BaseMessagePresenter<BubbleViewT, ViewModelBuilderT, InteractionHandl
         cell.performBatchUpdates({ () -> Void in
             self.messageViewModel.showsTail = decorationAttributes.canShowTail
             self.messageViewModel.showsAvatar = decorationAttributes.canShowAvatar
+            self.messageViewModel.isShowingSelectionIndicator = decorationAttributes.isShowingSelectionIndicator
```

Comment:
the way how you copy all the attributes from one entity to another looks a little bit suspicious. why don't you have the dedicated entity which represents the whole set of properties? in this way copying would look like: self.mesageViewModel.attributes = decorationAttributes.attributes